regifter.sql

```diff
@@ -9,110 +9,104 @@ CREATE DATABASE regifter;
 
 --
 \echo Create a table called gifts
--- with the following columns
--- id serial primary KEY
--- gift - string
--- giver - string
--- value - integer
--- previously_regifted boolean
+CREATE TABLE gifts ( id SERIAL PRIMARY KEY, gift TEXT, giver TEXT, price INT, previously_regifted BOOL);
 
 
 -- 
 \echo See details of the table you created
 -- 
-
+\d 
+or
+\dt
 
 -- 
 \echo Alter the table so that the column price is changed to value 
 -- 
-
+ALTER TABLE gifts RENAME price TO value;
 
 -- 
 \echo Insert a peach candle, given by 'Santa' thats value is 9 and has been previously regifted
 -- 
-
+INSERT INTO gifts (gift, giver, value, previously_regifted) VALUES ('peach candles', 'Santa', 9, TRUE);
 
 --
 \echo Query for all the columns in your gifts table
 -- 
-
+SELECT * FROM gifts;
 
 --
 \echo Uncomment below to insert 5 more gifts
 -- 
-
--- INSERT INTO gifts (gift, giver, value, previously_regifted)
--- VALUES
--- ('peach candle', 'Santa', '9', TRUE),
--- ('cinnamon candle', 'Nick', '19', TRUE),
--- ('soap on a rope', 'Rudolf', '29', FALSE),
--- ('potpurri', 'Elf on the Shelf', '39', TRUE),
--- ('mango candle', 'The Boss', '49', FALSE)
--- ;
+INSERT INTO gifts (gift, giver, value, previously_regifted)                      
+VALUES                                          ('mango candles', 'Santa', 4,FALSE),                                         ('coal', 'Santa', 2, TRUE),                                          ('telescope', 'Mother', 6, FALSE),                                        ('apple candles', 'Santa', 2, FALSE), 
```

Comment:
notice how in a gh commit these lines become harder to read?

Get in the habit of splitting these larger SQL queries across multiple lines.